### PR TITLE
Mario/deps update

### PR DESCRIPTION
**Pull request by blamario**
Updated `plutus-extra, `plutus-apps`, and all that follows.

### CODE REVIEW

**Comment by samuelWilliams99** [comment]
Is there a reason we're change to mlabs plutus-extra over Liqwid?

File: cabal.project [hunk 1]
```diff
@@ -1,4 +1,4 @@
-index-state: 2021-08-14T00:00:00Z
+-- index-state: 2021-08-14T00:00:00Z
```
Comment (samuelWilliams99) [inline cabal.project:1]: Can we not just update this to a more recent time over removing it?

Comment (blamario) [reply cabal.project:1]: Actually we don't even need to bump the date, I just tried to apply Las's suggestion and forgot to revert.

**Comment by blamario** [comment]
> Is there a reason we're change to mlabs plutus-extra over Liqwid?

Yes, it's https://github.com/Liqwid-Labs/plutus-extra/pull/192
Once that PR is merged or `plutus-extra` updated wholesale we can switch back to Liqwid.

File: cabal.project (hunk 1 above)
Comment (blamario) [reply cabal.project]: Reverted now.

**Comment by samuelWilliams99** [comment]
Gotcha, we can merge then but let's get a ticket up to revert to regular plutus-extra once we're able

**Review by samuelWilliams99 — APPROVED** [review]

**Pull request merged**